packages/core/src/BaseCheckBox/BaseCheckBox.styles.ts

```diff
@@ -42,36 +46,33 @@ export const { staticClasses, useClasses } = createClasses("HvBaseCheckBox", {
   icon: {},
   checked: {
     "& svg": {
-      border: `1px solid ${theme.colors.text}`,
-      backgroundColor: theme.colors.text,
+      border: `1px solid ${theme.colors.primaryStrong}`,
+      backgroundColor: theme.colors.primaryStrong,
```

Comment:
this affects DS3 😞

Reply:
Well, that's why we have theme overrides 👀